static/openapi.yaml

```diff
@@ -276,7 +465,32 @@ paths:
               schema:
                 type: array
                 items: {}
-          description: ''
+          description: 'ExplanationOfBenefit FHIR resources bundle'
+        '400':
+          content:
+            application/json:
+              schema:
+                type: array
+                items: {}
+          description: 'Error: Bad Request, e.g. the _lastUpdated operator is not valid.'
+        '401':
+          content:
+            application/json:
+              schema:
+                type: array
+                items: {}
+          description: 'Error: Unauthorized, e.g. Authentication credentials were not provided.'
+        '502':
+          content:
+            application/json:
+              schema:
+                type: array
+                items: {}
+            application/fhir+json:
```

Comment:
Just curious - why are there multiple content types here? Both `application/json` and `application/fhir+json`?

Reply:
there are cases where the fhir Error (OperationOutcome resource - app + json) fragment of it is checked and extracted as part of bb2 response (502) so the content type, but making it json only does not impact any thing here.., it's good to just use json.